test/LocalField/Completions.jl

```diff
@@ -39,6 +39,14 @@ end
   @test precision(bb) >= 20
 end
 
+@testset "Issue 1509" begin
+  F, _ = cyclotomic_field(3)
+  OF = maximal_order(F);
+  K, toK = completion(F, 2*OF);
+  setprecision!(toK, 100)
+  @test precision(toK(F(1))) == 100
```

Comment:
This test doesn't trigger the issue.
The (default) precision of `toK` is 64 and the issue only appears if the new precision is lower than the old one.

Reply:
ok will fix it